.github/workflows/sonarqube.yml

```
@@ -18,10 +18,61 @@
+            echo "FIREBASE_AUTH_PROVIDER_X509_CERT_URL=${{ secrets.FIREBASE_AUTH_PROVIDER_X509_CERT_URL }}"
+            echo "FIREBASE_CLIENT_X509_CERT_URL=${{ secrets.FIREBASE_CLIENT_X509_CERT_URL }}"
+            echo "FIREBASE_UNIVERSE_DOMAIN=${{ secrets.FIREBASE_UNIVERSE_DOMAIN }}"
+            
+          } > backend/.env
+        shell: bash
+
+      - name: Run tests and generate coverage report
+        run: |
+          cd backend
+          pytest --cov=api --cov=backend --cov-report=xml:coverage.xml
       
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)